CONTRIBUTING.md

```diff
@@ -84,17 +84,17 @@ You may type this line on your own when writing your commit messages. However, i
 
 ## Changelog
 
-OpenSearch-Java maintains version specific changelog by enforcing a change to the ongoing [CHANGELOG](CHANGELOG.md) file adhering to the [Keep A Changelog](https://keepachangelog.com/en/1.0.0/) format.
+OpenSearch-Java maintains a version specific changelog by enforcing a change to the ongoing [CHANGELOG](CHANGELOG.md) file adhering to the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) format.
```

Comment:
The project is Opensearch-java (capital O because start of sentence).

Version-specific I think has a `-`.